src/components/icon/icon.pw.tsx

```diff
@@ -0,0 +1,538 @@
+
+  colorData.forEach((tooltipFontColor) => {
+    test(`should pass accessibility tests when tooltip font color is set as ${tooltipFontColor}`, async ({
+      mount,
+      page,
+    }) => {
+      await mount(<IconTooltipComponent tooltipFontColor={tooltipFontColor} />);
+
+      await checkAccessibility(page, "color-contrast");
+    });
+  });
+
+  testData.forEach((tooltipId) => {
+    test(`should pass accessibility tests when tooltipId is set as ${tooltipId}`, async ({
+      mount,
+      page,
+    }) => {
+      await mount(<IconTooltipComponent tooltipId={tooltipId} />);
+
+      await checkAccessibility(page);
+    });
+  });
+
+  (["error", "add", "admin", "alert"] as IconProps["type"][]).forEach(
+    (iconType) => {
+      test(`should pass accessibility tests when iconType is set as ${iconType}`, async ({
+        mount,
+        page,
+      }) => {
+        await mount(<IconComponent type={iconType} />);
+
+        await checkAccessibility(page);
+      });
+    }
+  );
+
+  [true, false].forEach((boolVal) => {
+    test(`should pass accessibility tests icon color when disabled prop is set as ${boolVal}`, async ({
+      mount,
+      page,
+    }) => {
+      await mount(<IconComponent disabled={boolVal} />);
+
+      await checkAccessibility(page);
+    });
+  });
+
+  colorData.forEach((backgroundColor) => {
+    test(`should pass accessibility tests when background color is set as ${backgroundColor}`, async ({
+      mount,
+      page,
+    }) => {
+      await mount(<IconComponent bg={backgroundColor} />);
+
+      await checkAccessibility(page);
+    });
+  });
+
+  colorData.forEach((iconColor) => {
+    test(`should pass accessibility tests when icon color is set as ${iconColor}`, async ({
+      mount,
+      page,
+    }) => {
+      await mount(<IconComponent color={iconColor} />);
+
+      await checkAccessibility(page);
+    });
+  });
+
+  [SIZE.SMALL, SIZE.MEDIUM, SIZE.LARGE, SIZE.EXTRALARGE].forEach((fontSize) => {
+    test(`should pass accessibility tests when fontSize is set as ${fontSize}`, async ({
+      mount,
+      page,
+    }) => {
+      await mount(<IconComponent fontSize={fontSize} />);
+
+      await checkAccessibility(page);
+    });
+  });
+
+  (["circle", "rounded-rect", "square"] as IconProps["bgShape"][]).forEach(
+    (bgShape) => {
+      test(`should pass accessibility tests when bgShape is set as ${bgShape}`, async ({
+        mount,
+        page,
+      }) => {
+        await mount(<IconComponent bgShape={bgShape} />);
+
+        await checkAccessibility(page);
+      });
+    }
+  );
+
+  [
+    SIZE.EXTRASMALL,
+    SIZE.SMALL,
+    SIZE.MEDIUM,
+    SIZE.LARGE,
+    SIZE.EXTRALARGE,
+  ].forEach((size) => {
+    test(`should pass accessibility tests when bgSize is set as ${size}`, async ({
+      mount,
+      page,
+    }) => {
+      await mount(<IconComponent bgSize={size} />);
+
+      await checkAccessibility(page);
+    });
+  });
+
+  ([
+    [["left"], "left", "bottom"],
```

Comment:
**suggestion:** Same here. The syntax suggested previously _should_ work here too.

Reply:
Ignore this. flipOveride expects an array 😶‍🌫️